source/reference/minio-mc/mc-license-register.rst

```diff
@@ -110,10 +104,21 @@ Parameters
    If the deployment is airgapped, but the local device where you are using the :ref:`minio client <minio-client>` has network access, you do not need to use the ``--airgap`` flag.
 
 
-
 Global Flags
 ~~~~~~~~~~~~
 
 .. include:: /includes/common-minio-mc.rst
    :start-after: start-minio-mc-globals
    :end-before: end-minio-mc-globals
+
+Behavior
+--------
+
+.. versionchanged:: RELEASE.2023-01-18T04-36-38Z
```

Comment:
move this down to line 122

Reply:
can also do 'version added' since this is a new feature.

Reply:
Done.